### PR TITLE
Embed Site

### PR DESCRIPTION
**Pull request by joern-vh**
Added Data to Map Widget (Solves #99).
Contributors now alphabetic sorted (Solves #119).
Attributions updated (Solves #115).

### CODE REVIEW

File: client/components/static/Attribution.js [hunk 1]
```diff
@@ -74,6 +74,21 @@ class Attribution extends React.Component {
...
+              <td>
+                Icons used in our Widgets.
+              </td>
+              <td>
+                <a href='http://www.flaticon.com/authors/dave-gandy' target='_blank'>Icon for marking external Links</a> <br />
+                <a href='http://www.flaticon.com/authors/dave-gandy' target='_blank'>Full-screen icon</a> <br />
+                <a href='http://www.flaticon.com/authors/dave-gandy' target='_blank'>Minimize icon</a> <br />
+                <a href='http://www.flaticon.com/authors/dave-gandy' target='_blank'>Download icon</a> <br />
+                <a href='http://www.flaticon.com/authors/dave-gandy' target='_blank'>Flag icon</a> <br />
+                <a href='http://www.flaticon.com/authors/dave-gandy' target='_blank'>Source icon</a> <br />
```
Comment (kordianbruck) [inline client/components/static/Attribution.js:88]: Not sure, we actually need all icons individually in here but who cares.

File: client/components/About.js [hunk 2]
```diff
@@ -35,52 +35,53 @@ class About extends React.Component {
...
                     <ul>
-                      <li><a href='https://www.linkedin.com/in/gyachdav'>Guy Yachdav</a></li>
+                      <li><a href='https://www.linkedin.com/in/burkhard-rost-b3ba9449/?ppe=1'>Burkhard Rost</a></li>
                       <li><a href='https://github.com/sacdallago'>Christian Dallago</a></li>
+                      <li><a href='https://www.linkedin.com/in/gyachdav'>Guy Yachdav</a></li>
                       <li><a href='https://bruck.me'>Kordian Bruck</a></li>
-                      <li><a href='https://github.com/vviro'>Vladimir Viro</a></li>
                       <li><a href='https://github.com/pfent'>Philipp Fent</a></li>
                       <li><a href='https://github.com/goldbergtatyana'>Tatyana Goldberg</a></li>
+                      <li><a href='https://github.com/vviro'>Vladimir Viro</a></li>
```
Comment (kordianbruck) [inline client/components/About.js:44]: Sorting by first name also works I guess.

**Review by kordianbruck — APPROVED** [review]

File: client/components/About.js (hunk 2 above)
Comment (joern-vh) [reply client/components/About.js:44]: Should I Change to lastname?

File: client/components/static/Attribution.js (hunk 1 above)
Comment (joern-vh) [reply client/components/static/Attribution.js:88]: As discussed in #115 we were supposed to list all of them.

File: client/components/About.js (hunk 2 above)
Comment (pfent) [reply client/components/About.js:44]: @kordianbruck just wants to be no. 1 😆

Comment (kordianbruck) [reply client/components/About.js:44]: pssst :dancer:

**Pull request merged**